core/src/main/scala/com/phaller/rasync/HandlerPool.scala

```diff
@@ -444,4 +449,20 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
 
   def reportFailure(t: Throwable): Unit =
     t.printStackTrace()
+
+  /**
+   * Interrupt the computation of cells. It can be resumed using the `resume` method.
+   */
+  def interrupt(): Unit = {
```

Comment:
"interrupt" -> "suspend". The pair suspend/resume is much more common. Moreover, `interrupt` already has a different meaning for threads on the JVM (including `InterruptedException` etc.), so it is actually quite confusing to call this method `interrupt`!

Reply:
Changed to `suspend`.